.github/workflows/simorgh-misc-checks.yml

```diff
@@ -39,6 +39,33 @@ jobs:
         if: steps.cache.outputs.cache-hit != 'true'
         run: ./scripts/installNodeModules.sh
 
+      - name: Get Storybook Branch Permalink
+        uses: actions/github-script@v6
+        id: get-storybook-permalink
+        with:
+          script: |
+            const storybookBranch = '${{ github.event.pull_request.head.ref }}'.toLowerCase().replaceAll(/[/.]/g, '-').trim('-').substring(0, 37);
+            const storybookPermalink = `https://${storybookBranch}--5d28eb3fe163f6002046d6fa.chromatic.com`;
+            console.log(`Storybook Permalink is: ${storybookPermalink}`);
```

Comment:
Just a wee console.log that could be removed?

Reply:
Ah good shout, I had added it so that I could see it in the build output, but we can see it on the PR, and in the "Publish Storybook Branch Permalink" step, so I'll remove now. Thanks! 🙌